### PR TITLE
Add Amazon Fire TV YouTube remote control

### PR DESCRIPTION
**Pull request by wvu**
Wireshark is my co-pilot. :D

```
msf auxiliary(firetv_youtube) > run

[+] Playing https://www.youtube.com/watch?v=HkhSZyYmpO4
[*] Auxiliary module execution completed
msf auxiliary(firetv_youtube) > set action Stop 
action => Stop
msf auxiliary(firetv_youtube) > run

[*] Stopping video
[*] Auxiliary module execution completed
msf auxiliary(firetv_youtube) > 
```

Pretty tired of 10 hours of Epic Sax Guy. :)

### CODE REVIEW

**Comment by wvu-r7** [comment]
Travis failure unrelated, btw.

**Comment by todb-r7** [comment]
@limhoff-r7 might want to look at https://travis-ci.org/rapid7/metasploit-framework/jobs/51069739#L1586 , kicking off a new build for that now.

File: modules/auxiliary/admin/firetv/firetv_youtube.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      Opt::RPORT(8008),
+      OptString.new('VID', [true, 'Video ID', 'HkhSZyYmpO4'])
+    ])
+  end
+
+  def run
+    case action.name
+    when 'Play'
+      stop
+      Rex.sleep(1)
```
Comment (todb-r7) [inline modules/auxiliary/admin/firetv/firetv_youtube.rb]: `Kernel#sleep` is the same as `Rex#sleep` so it's okay to just `sleep`.

**Comment by todb-r7** [comment]
If you agree with the assessment of sleep I'm happy to merge the change here and land away, @wvu-r7

**Comment by wvu-r7** [comment]
@todb-r7: We should probably update the docs, then, since we've been recommending the use of ```Rex.sleep```. I understand that it's just ```sleep```, so the change is fine by me!

**Comment by wvu-r7** [comment]
Oh, I think that was in ```HACKING```, which is gone now. Nevermind!

**Comment by todb-r7** [comment]
I wonder why 45b16c9 is being marked as a failure. The error reported is:

```
$ cd rapid7/metasploit-framework
3.21s$ git fetch origin +refs/pull/4775/merge:
fatal: Couldn't find remote ref refs/pull/4775/merge
The command "eval git fetch origin +refs/pull/4775/merge:" failed. Retrying, 2 of 3.
fatal: Couldn't find remote ref refs/pull/4775/merge
The command "eval git fetch origin +refs/pull/4775/merge:" failed. Retrying, 3 of 3.
fatal: Couldn't find remote ref refs/pull/4775/merge
The command "eval git fetch origin +refs/pull/4775/merge:" failed 3 times.
The command "git fetch origin +refs/pull/4775/merge:" failed and exited with 128 during .
Your build has been stopped.
```

PR #4775 is definitely real...

**Pull request merged**